src/MessagingThreadHistory/MessageMetadata.less

```diff
@@ -5,6 +5,8 @@
   flex-direction: column;
   .margin--y--m();
   .margin--x--xs();
+  // Without this, messages 'bunch up' on Safari.
+  flex-shrink: 0;
```

Comment:
Just wondering - do we still need it on each of the inner containers now that we have this outer container? `center, right, left`

Reply:
Oh good point, I'll test it out locally!